frontend/package.json

```diff
@@ -30,8 +30,8 @@
     "js-cookie": "2.2.1",
     "jwt-decode": "3.1.2",
     "lottie-web": "5.7.8",
-    "next": "11.1.2",
-    "next-translate": "1.1.1",
+    "next": "12.0.7",
+    "next-translate": "^1.1.1",
```

Comment:
As we discussed in the PR https://github.com/Stooa/Stooa/pull/110, for the moment we are not using carets  :)